shaolinmkz/iReporter
# #162339296 Build out endpoint to update red flag location and comment

#### What does this PR do?
Have the update red flag location and comment API functional

#### Description of Task to be completed?
Having the endpoint  below working:
* PATCH `host:port/api/v1/red-flags/<red-flag-Id>/location`
* PATCH `host:port/api/v1/red-flags/<red-flag-Id>/comment`

#### How should this be manually tested?
After cloning the repo, cd into it then:
* RUN `npm test` to see test result
* RUN `npm start` to start the server

Using Postman test the endpoint below:
* PATCH `host:port/api/v1/red-flags/<red-flag-Id>/location`
* PATCH `host:port/api/v1/red-flags/<red-flag-Id>/comment` with the header:
_key:_ Content-Type value: application/json

#### What are the relevant pivotal tracker stories?
[#162339296](https://www.pivotaltracker.com/story/show/162339296)
